src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,40 @@
+import { useContext } from 'react';
+import { TodosContext } from '../../context/TodosContext';
+import { TodosFilter } from '../TodosFilter';
+
+type Props = {
+
+};
+
+export const Footer: React.FC<Props> = () => {
+  const {
+    todos,
+    setTodos,
+    status,
+  } = useContext(TodosContext);
+
+  const clearDoneTodos = () => {
+    setTodos(todos.filter(todo => todo.completed === false));
+  };
+
+  return (
+    todos.length ? (
```

Comment:
```suggestion
  if(todos.length) {
    return null
  }

  return (
```
it looks much better (you can also move that condition in App.tsx)